tree/tree/inc/TBasket.h

```diff
@@ -52,6 +51,8 @@ class TBasket : public TKey {
    Int_t       fNevBuf;          ///< Number of entries in basket
    Int_t       fLast;            ///< Pointer to last used byte in basket
    Bool_t      fHeaderOnly;      ///< True when only the basket header must be read/written
+   UChar_t fIOBits{
+      0}; ///<!IO feature flags.  Serialized in custom portion of streamer to avoid forward compat issues unless needed.
```

Comment:
Please align the variable names :)

Reply:
Should I listen to you or to `clang-format` here?

Reply:
Nope :)  It can't do what we need here ...